modules/core/include/opencv2/core/hal/intrin.hpp

```diff
@@ -433,6 +433,29 @@ template <> struct V_RegTrait128<double> {
 };
 #endif
 
+inline unsigned int trailingZeros32(unsigned int value) {
```

Comment:
Could you remove duplicate of this function from [`contours.cpp`](https://github.com/opencv/opencv/blob/17eef4d8a967a3541ebeefa44b80cf8001b0d5a5/modules/imgproc/src/contours.cpp#L55) file and use this one?